json-endpoints.php

```diff
@@ -2493,6 +2493,8 @@
 		'jetpack_testimonial_posts_per_page'   => '(int) Number of testimonials to show per page',
 		'jetpack_portfolio'                    => '(bool) Whether portfolio custom post type is enabled for the site',
 		'jetpack_portfolio_posts_per_page'     => '(int) Number of portfolio projects to show per page',
+		'advanced_seo_front_page_description'  => '(string) The seo meta description for the site.',
```

Comment:
In that case I would prefer to keep the existing names that are used on WordPress.com for now. Changing them would cause issues for users that have already set those settings, and we would have to handle those cases somehow.